src/release_linux.go

```diff
@@ -16,7 +20,14 @@ func writeDesktopFiles(mc *metaConfig) {
 }
 
 func configureAutologin(mc *metaConfig) {
-	// set up user autologin
+	if _, err := os.Stat("/usr/share/lightdm/lightdm.conf.d/50-ubuntu.conf"); err == nil {
+		shellCommand("echo autologin-user=" + mc.Config.User + " > /usr/share/lightdm/lightdm.conf.d/50-ubuntu.conf")
+	} else if  _, err := os.Stat("/etc/gdm/custom.conf"); err == nil {
+		shellCommand("echo AutomaticLogin=True > /etc/gdm/custom.conf")
+		shellCommand("echo AutomaticLogin=" + mc.Config.User + " > /etc/gdm/custom.conf")
+	} else {
+		failPrint("Autologin not able to configure on this system")
```

Comment:
Should it exit here @sourque?

Reply:
Nah I wouldn't think so, as long as we alert the user they can just edit it themselves, since we can't guarantee they're using LightDM or GDM3 we wouldn't want to prevent them from using the rest of the release function.